sql-streaming-copier/README.md

```diff
@@ -0,0 +1,127 @@
+# Pub/Sub Copy Pipeline
+
+A dataflow template which performs copies to and from streaming systems using
+beam SQL.
+
+This can perform copies from any of:
+1) Cloud Pub/Sub
+2) Pub/Sub Lite
+3) Kafka
+
+This can perform copies to any of:
+1) Cloud Pub/Sub
+2) Pub/Sub Lite
+3) Kafka
+4) BigQuery
+
+All sources and sinks should have a simple SQL transform to/from the following
```

Comment:
I would structure this slightly differently. I'm worried that readers will think that they need to implement all of this to use the built-in transforms. How about we first tell people how to run this for the built in source/sinks, and then we add something like the following:

Additional Sinks/Sources can be added by implementing StandardSink (link) and StandardSource (link), which should parse data from/to a generic data model matching the following schema: